src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,49 +1,128 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_SIZE = 10;
+    private static final double INCREASING_COEFFICIENT = 10;
+    private static final int IMPORTANT_VALUE = 1;
+
+    private T[] elementData = (T[]) new Object[DEFAULT_SIZE];
+    private int size = 0;
+
+    private T[] grow(int oldSize) {
```

Comment:
private methods should be placed after public

Reply:
done